### ISSUE TITLE
Missing default modifier for TSNamespaceExportDeclaration

### ISSUE THREAD

**Issue by vjeux** [eslint/typescript-eslint-parser#288]
**What version of `typescript-eslint-parser` are you using?**
b7220fdbf5a923c64bb6e5bbbfc2ad689ec779eb

**What code were you trying to parse?**

```js
declare module 'react-native-video' {
  export default class Video { }
}
```

**What did you expect to happen?**

Have a way to know that it's `default`

**What happened?**

There is no such thing

```js
{ type: 'TSNamespaceExportDeclaration',
  declaration:
   { type: 'ClassDeclaration',
     range: [ 55, 70 ],
     loc: { start: [Object], end: [Object] },
     id:
      { type: 'Identifier',
        range: [Object],
        loc: [Object],
        name: 'Video' },
     body: { type: 'ClassBody', body: [], range: [Object], loc: [Object] },
     superClass: null,
     implements: [],
     decorators: [] },
  range: [ 40, 70 ],
  loc: { start: { line: 2, column: 2 }, end: { line: 2, column: 32 } } }
```

and prettier prints it as

```js
declare module "react-native-video" {
  export class Video {}
}
```

**Comment by soda0289** [comment]
Thanks for the report

This looks like a duplicate of https://github.com/eslint/typescript-eslint-parser/issues/263
I think its better to just use the ESTree export node types and have a PR that just needs a final review #265

**Comment by vjeux** [comment]
Thanks! This is the only blocker left on this project side to correctly print many real typescript projects:

![image](https://cloud.githubusercontent.com/assets/197597/26382044/1088b050-3fde-11e7-8b77-500b6c96ad25.png)

https://github.com/prettier/prettier/issues/1643#issuecomment-303478032

**Comment by soda0289** [comment]
Fixed in master
